javascript/javascript3/week2/promiseSetTime.js

```diff
@@ -0,0 +1,10 @@
+async function promiseResolve(resolveAfter) {
```

Comment:
There is really no need to use the async prefix here. async is only used if you want to use await inside the function